fujiji-server/migration-scripts/example.sql

```diff
@@ -1,4 +1,4 @@
-USE fujiji
+USe fujiji
```

Comment:
I assume this is an accidental change, can you change it back to `USE`?

Reply:
I just fixed it